Dockerfile

```diff
@@ -193,13 +198,6 @@ USER guacd
 # Expose the default listener port
 EXPOSE 4822
 
-# Start guacd, listening on port 0.0.0.0:4822
-#
-# Note the path here MUST correspond to the value specified in the
-# PREFIX_DIR build argument.
-#
-CMD /opt/guacamole/sbin/guacd -b 0.0.0.0 -L $GUACD_LOG_LEVEL -f
```

Comment:
Why has this been removed  ?

Reply:
It confused me, when I was messing with log versions. 

We actually call guacd from guacd.sh